components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,116 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumbs extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				--d2l-breadcrumbs-background-color: rgb(251, 252, 252);
```

Comment:
I couldn't find a reference to this variable being overridden anywhere through code search. Can we just remove it and hardcode the colour wherever it's used? Just not something we need to support forever if it doesn't need to be extensible.

Reply:
Removed ✔️